snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -180,16 +184,65 @@ open class Snapshot: NSObject {
     }
 }
 
-extension XCUIElement {
-    var isLoadingIndicator: Bool {
-        let whiteListedLoaders = ["GeofenceLocationTrackingOn", "StandardLocationTrackingOn"]
-        if whiteListedLoaders.contains(self.identifier) {
-            return false
+private extension XCUIElementAttributes {
+    var isNetworkLoadingIndicator: Bool {
+        let oldLoadingIndicatorSize = CGSize(width: 10, height: 20)
+        
+        let hasOldLoadingIndicatorSize = frame.size == oldLoadingIndicatorSize
+        let hasNewLoadingIndicatorSize = frame.size.width.isBetween(46, and: 47) && frame.size.height.isBetween(2, and: 3)
+        
+        return !hasWhiteListedIdentifier && (hasOldLoadingIndicatorSize || hasNewLoadingIndicatorSize)
+    }
```

Comment:
I think this might be a bit more readable with `guard` and `switch`, let me know what you think:

```swift
var isNetworkLoadingIndicator: Bool {
    guard !hasWhiteListedIdentifier else { return false }

    switch frame.size {
    case CGSize(width: 10, height: 20):
        // old loading indicator
        return true
    case _ where frame.size.width.isBetween(46, and: 47) && frame.size.height.isBetween(2, and: 3):
        // iPhone X loading indicator
        return true
    default:
        return false
    }
}
```

Reply:
I agree with the early exit through a `guard` or `if`, then I don't even need to negate `hasWhiteListedIdentifier`. That's a good catch

But the switch has a cyclomatic complexity of 4 and needs comments, whereas when I write 
```swift
let hasOldLoadingIndicatorSize = frame.size == CGSize(width: 10, height: 20)
let hasNewLoadingIndicatorSize = frame.size.width.isBetween(46, and: 47) && frame.size.height.isBetween(2, and: 3)

return hasOldLoadingIndicatorSize || hasNewLoadingIndicatorSize
```
it has a cyclomatic complexity of 2 and it doesn't need comments because of named constants ;)

Reply:
Fair :+1: